src/ngx_http_lua_exten.c

```diff
@@ -0,0 +1,34 @@
+
+/*
+ * Copyright (C) Yichun Zhang (agentzh)
+ */
+
+
+#ifndef NGX_LUA_NO_FFI_API
+
+
+#ifndef DDEBUG
+#define DDEBUG 0
+#endif
+#include "ddebug.h"
+
+
+#include "ngx_http_lua_util.h"
+
+
+char *
+ngx_http_lua_ffi_req_get_ext(ngx_http_request_t *r, size_t *len)
+{
+    if (r->connection->fd == (ngx_socket_t) -1) {
+        return NULL;
```

Comment:
@agentzh  Yeah, changed, and also https://github.com/openresty/lua-resty-core/pull/105.